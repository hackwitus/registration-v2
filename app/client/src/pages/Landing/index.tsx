import * as React from 'react';
import { useAuth0 } from '../../auth/auth0-spa';
import Navbar from '../../components/NavBar';

const Landing: React.FC = () => {
  const { isAuthenticated, loading, user, logout, loginWithRedirect } = useAuth0();
  return (
    <>
      <Navbar
        user={user}
        loading={loading}
        isAuthenticated={isAuthenticated}
        logout={logout}
        login={loginWithRedirect}
      />
    </>
  );
};

export default Landing;
