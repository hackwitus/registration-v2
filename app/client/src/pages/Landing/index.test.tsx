import * as React from 'react';
import { shallow } from 'enzyme';
import Landing from './index';

describe('Landing Page Component', () => {
  it('renders landing page', () => {
    const landing = shallow(<Landing />);
    expect(shallow).toMatchSnapshot();
  });
});
