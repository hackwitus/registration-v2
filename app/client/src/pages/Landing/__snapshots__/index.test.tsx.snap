// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component renders landing page 1`] = `[Function]`;

exports[`Landing renders landing page 1`] = `[Function]`;

exports[`renders landing page 1`] = `[Function]`;
